contrib/cmd/k8s-broker/k8s-broker.go

```diff
@@ -55,12 +55,14 @@ func main() {
 	}
 
 	if len(options.HelmBinary) == 0 {
-		panic("Need helm_binary specified")
+		fmt.Fprintln(os.Stderr, "--helm_binary must be specified")
+		os.Exit(1)
```

Comment:
perhaps, but I'd really prefer if we didn't "log" command line errors. I think the output would look something like this:
```
E0121 15:46:37.334798  127442 controller.go:47] --helm_binary must be specified
```
and I don't find the user friendly. To me there's a difference between stuff written to a log file and stuff for end-user consumption - and stuff for end users doesn't usually need all of the machine readable stuff prefixing it.